lib/authentication/view/sign_up_view.dart

```diff
@@ -198,12 +139,10 @@ class _SignUpViewState extends State<SignUpView> {
         }
 
         fields[S.of(context).labelEmail] += S.of(context).stringEmailDomain;
-        nodes.asMap().forEach((i, node) {
-          if (i > 0) {
-            fields[filter.localizedLevelNames[i - 1]
-                [LocaleProvider.localeString]] = node.name;
-          }
-        });
+        if(dropdownController.path != null) {
```

Comment:
Don't forget to ctrl+alt+L on modified files before committing them.
```suggestion
        if (dropdownController.path != null) {
```